specification/implementation/metadata/general-requirements/index.md

```diff
@@ -64,3 +64,6 @@ In order to guarantee interoperability, all references within a CSIP Information
 
 ### Referencing other packages
 It is important that external references to related packages, like internal references, are expressed consistently. All external references MUST USE `mets/@OBJID` attribute of the package.
+
+### Examples
+Through out this specification there are fictive examples used. This means that the examples uses dvalues from value lists defined in CSIP as well as the elements described through out CSP but other given values are ficitive and dont constititure a real life example. More elaborate examples will be made available through guidelines that will be published.
```

Comment:
This document illustrates use of the specification using generic examples These examples use values from vocabularies and elements defined by the CSIP to demonstrate some specific uses of the specification. Outside these CSIP specific values the METS values are fictive and are not meant to constitute real life examples. More complete examples will be provided in the CSIP user guidelines that will be published at a future date.